src/React/App.js

```diff
@@ -1,23 +1,18 @@
-import logo from './logo.svg';
 import './App.css';
+import React from "react";
+import Navbar from "react-bootstrap/Navbar";
+import Routes from "./Routes";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="App container py-3">
+      <Navbar collapseOnSelect bg="light" expand="md" className="mb-3">
+        <Navbar.Brand className="font-weight-bold text-muted">
+          Calgary Distress Centre
+        </Navbar.Brand>
+        <Navbar.Toggle />
+      </Navbar>
```

Comment:
Typically navbars wouldn't be displayed on a login page.

Can we undo this since navbar will need more thought from design team and isn't needed for this login page PR.